client/src/components/profile/components/heat-map.tsx

```diff
@@ -84,48 +84,28 @@ class HeatMapInner extends Component<HeatMapInnerProps, HeatMapInnerState> {
   }
 
   render() {
-    const { calendarData, currentStreak, longestStreak, pages, t } = this.props;
+    const { calendarData, currentStreak, longestStreak, pages, t, points } =
+      this.props;
     const { startOfCalendar, endOfCalendar } = pages[this.state.pageIndex];
-    const title = `${startOfCalendar.toLocaleDateString([localeCode, 'en-US'], {
-      year: 'numeric',
-      month: 'short'
-    })} - ${endOfCalendar.toLocaleDateString([localeCode, 'en-US'], {
-      year: 'numeric',
-      month: 'short'
-    })}`;
     const dataToDisplay = calendarData.filter(
       data => data.date >= startOfCalendar && data.date <= endOfCalendar
     );
 
     return (
       <FullWidthRow>
-        <Row className='heatmap-nav'>
-          <button
-            className='heatmap-nav-btn'
-            disabled={!pages[this.state.pageIndex - 1]}
-            // eslint-disable-next-line @typescript-eslint/unbound-method
-            onClick={this.prevPage}
-            style={{
-              visibility: pages[this.state.pageIndex - 1] ? 'unset' : 'hidden'
-            }}
-          >
-            &lt;
-          </button>
-          <span>{title}</span>
-          <button
-            className='heatmap-nav-btn'
-            disabled={!pages[this.state.pageIndex + 1]}
-            // eslint-disable-next-line @typescript-eslint/unbound-method
-            onClick={this.nextPage}
-            style={{
-              visibility: pages[this.state.pageIndex + 1] ? 'unset' : 'hidden'
-            }}
-          >
-            &gt;
-          </button>
-        </Row>
         <Spacer />
-
+        <Row>
```

Comment:
The streak labels and their associated values should probably not be separated like this as it removes the association between the label and its value for screen reader users. There are several ways we could do this. This looks like tabular data but I think a table might be overkill here. I think this might be a good place for a `dl` element.